frontend/app/components/address.tsx

```diff
@@ -18,14 +18,14 @@ function formatAddress(address: string, city: string, country: string, provinceS
   // prettier-ignore
   const lines = [
     formattedAddress,
-    `${city}${provinceState ? ` ${provinceState}` : ''}${postalZipCode ? `  ${postalZipCode}` : ''}`,
+    `${city},${provinceState ? ` ${provinceState}` : ''}${postalZipCode ? `  ${postalZipCode}` : ''}`,
```

Comment:
In case no `provinceState` is specified:

```suggestion
    `${city}${provinceState ? `, ${provinceState}` : ''}${postalZipCode ? `  ${postalZipCode}` : ''}`,
```

Reply:
Referring to @sebastien-comeau's comment, I noticed that there is a `lines` and `linesAlt`, where `lines` seems to follow the Canada Post address format (city, province/state, postal/ZIP code one a single line with no comma.

`linesAlt` seems to be another format possibly from designers/Figma. My suggestion would be to add the comma in `linesAlt` but keep `lines` without the comma, since the Canada Post address format doesn't have a comma.

Reply:
@nicholas-ly @sebastien-comeau Updated. I removed `altFormat={true}` so all the address is now using the Canada Post format. As Nick recommended, I kept the comma in the `lineAlt` format.

Reply:
Don't want to open a new can of worms but the Canada Post format should display the province/state abbreviation (ie. `ON` instead of `Ontario`). Currently the review pages show the full province/state name.

Also, if we're not using `linesAlt` anywhere, can we remove it from the `Address` component?

Feel free to overrule any of the above - just my thoughts.

(cc @sebastien-comeau)

Reply:
@nicholas-ly PR to use `abbr` with Address component.
https://github.com/DTS-STN/canadian-dental-care-plan/pull/1701